rtic/build.rs

```diff
@@ -16,7 +16,7 @@ fn main() {
         //even within the same target
         //need some other way to discern
     } else if target.starts_with("riscv32i") {
-        println!("cargo:rustc-cfg=feature=\"riscv-esp32c3\"");
```

Comment:
I'm sure you have some plan for these TODOs :)

Reply:
I'm not sure about the purpose of this `build.rs` file. Doesn't Cargo deal with all these feature flags?

Reply:
Ok, so I changed the approach in `build.rs`. Instead of analyzing the target, I check the active features and depending on the backend, I enable the additional feature.